docs/application/dotnet/guides/nui/xaml/TypeConverter.md

```diff
@@ -0,0 +1,66 @@
+# TypeConverters and XAML
+
+This topic introduces the purpose of type conversion from string as a general XAML language feature. Such as converter "Red" to <code>Color.Red</code>. If you write a custom class, and you want instances of your class to be usable as XAML settable attribute values, you might need to write a custom TypeConverter class.
+
+## Type Conversion Concepts
+
+When you set an attribute value in a XAML file, the initial type of that value is a string in pure text. Even other primitives such as Double are initially text strings to a XAML processor.
+
+If the value is a primitive(such as a numeric value), a direct conversion of the string is attempted. If the value is an enumeration, the string is used to check for a name match to a named constant in that enumeration. If the value is neither a parser-understood primitive nor an enumeration, then need a type converter class.
+
+For instance, Tizen.NUI defines some properties that take a value of type <code>Size2D</code>. A <code>Size2D</code> is a value that describes two-dimensional size, and it really just has two important properties: Width and Height. When you specify a size2D in XAML, you specify it as a string with a comma between the Width and Height values you provide. For example:
+
+``` xml
+<ImageView PositionX="10" PositionY="320" Size2D="300, 300" ResourceUrl="ImageResourcePath" />
+```
+
+In this case type converter is the class <code>Size2DTypeConverter</code>.
```

Comment:
Are we saying:
In this scenario, the type converter is the `Size2DTypeConverter` class.
If a type converter is not available, you must use verbose markup as shown in the following code:

Please verify so that I can rephrase it better.